clmm/utils.py

```diff
@@ -153,6 +153,8 @@ def compute_radial_averages(xvals, yvals, xbins, yerr=None, error_model='ste', w
         Indices of the bins (corresponding to `xbins`) in which each value
         of `xvals` belongs.  Same length as `yvals`.  A binnumber of `i` means the
         corresponding value is between (xbins[i-1], xbins[i]).
+    wts_sum: bumpy.array
+        Sums of weights in each bin
```

Comment:
Sums of weights in each bin --> Sum of individual galaxy weights in each radial bin?

Reply:
This is a generic statistical function, it does not concern galaxies specifically, but I made a minor update on this docstring.

Reply:
Yeap, right!